### DISCUSSION TITLE
Has anyone had an issue with SwipeView Interfering with TapGestureRecognizer on Android Devices?

### DISCUSSION THREAD

**Discussion by kellymmcd** [dotnet/maui#23565] [Q&A]
I'm encountering an issue with a SwipeView in my .NET MAUI application. I have a CollectionView that displays a list of appointments, allowing users to tap to open an appointment or swipe right or left to reveal buttons for editing or deleting the appointments. The setup is similar to the following:

```
<CollectionView ItemsSource="{Binding Items}">
    <CollectionView.ItemTemplate>
        <DataTemplate>
            <SwipeView>
                <SwipeView.RightItems>
                    <SwipeItems>
                        <SwipeItem Text="Delete"
                                        BackgroundColor="LightPink"/>
                    </SwipeItems>
                </SwipeView.RightItems>
                <SwipeView.LeftItems>
                    <SwipeItems>
                        <SwipeItem Text="Edit"
                                BackgroundColor="Orange"/>
                    </SwipeItems>
                </SwipeView.LeftItems>
                <SwipeView.Content>
                    <ContentView>
                        <Label Text="{Binding Title}"
                                   Padding="30"/>
                        <ContentView.GestureRecognizers>
                            <TapGestureRecognizer Command="{Binding Command}" CommandParameter="{Binding CommandParameter}"/>
                        </ContentView.GestureRecognizers>
                    </ContentView>
                </SwipeView.Content>
            </SwipeView>
        </DataTemplate>
    </CollectionView.ItemTemplate>
</CollectionView>
```

On an Android device, if the user moves their finger even slightly while tapping, it triggers the swipe action, making it difficult to trigger the tap gesture. This issue does not occur on an emulator when using mouse to click button.

Has anyone encountered a similar issue or found a workaround for this? Any suggestions would be greatly appreciated.

Thank you!

**Comment by RsZoli** [comment]
@kellymmcd Hey!
I have this exact behaviour on iOS, Android is working fine for me, but on iOS, it is impossible to trigger the tap event in this described case!
